.papr.yml

```diff
@@ -3,7 +3,7 @@ cluster:
     - name: testnode
       distro: fedora/26/atomic
   container:
-    image: registry.fedoraproject.org/fedora:25
+    image: registry.fedoraproject.org/fedora:27
```

Comment:
I'd just keep 26 here to make sure running from the f26 userspace is still OK. We're already testing 27 down there right?

Reply:
In the same vein, does it make sense to switch the `centos/7/atomic` contexts to use a CentOS base image?

Reply:
I'd say so yeah. More so than f26 vs f27. :)